### DISCUSSION TITLE
How to enter dashboard?

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [gravitational/teleport#10291] [Q&A] [answered]
Hi, I install Teleport with docker-compose and I enter login window on my local address but there's field to enter login,  password and totp key.  I didn't configure user, password and 2FA yet. 
It should be there I think. Some regular admin/admin and totp app configuration.
Regards.

**Comment by scottpgallagher** [comment, marked as answer]
You would need to enter the running container and add the user from there. I'm assuming your following this guide? https://goteleport.com/docs/getting-started/docker-compose/ If so then you would first need to ssh to the container `ssh root@luna.teleport` from there you can add a user with `tctl users add testuser --roles=editor,access --logins=root,ubuntu`

**Reply by diktamxx** [reply]
> You would need to enter the running container and add the user from there. I'm assuming your following this guide? https://goteleport.com/docs/getting-started/docker-compose/ If so then you would first need to ssh to the container `ssh root@luna.teleport` from there you can add a user with `tctl users add testuser --roles=editor,access --logins=root,ubuntu`

The link has become invalid.

**Comment by PrzemekSkw** [comment]
Thanks, I did it.
I'm not any programmer just want to have web dashboard to manage my home servers (like raspberry pi with nextcloud) and one VPs. Is Teleport good option to use it that way. To connect ssh and some file manager option to download/upload files? Because looks little complicated for me.
Regards.

**Reply by scottpgallagher** [reply]
Teleport can easily add SSH hosts into the dashboard and from the UI you can easily connect to those SSH systems. I'd highly recommend this youtube video that gives a great overview of the product from one of our engineers: https://youtu.be/_DUuHJKWD8I

**Comment by PrzemekSkw** [comment]
Thank You very much for help 😃
Regards.

**Comment by PrzemekSkw** [comment]
@scottpgallagher can You tell me why I have error when try to connect from my laptop to my server (where I install Teleport):
![Zrzut ekranu_20220211172615](https://user-images.githubusercontent.com/25219988/153629621-b4904298-5215-4997-9579-079b0822892d.png)

```
disconnected
Failed to launch: user: unknown user dietpi.
Process exited with status 255
```
When I choose root in `connect` down arrow I can connect but don't know where I'm connected:
```
root@820f7b697e1e:~# ls
root@820f7b697e1e:~# cd /home/dietpi
-bash: cd: /home/dietpi: No such file or directory
root@820f7b697e1e:~# cd /home
root@820f7b697e1e:/home# ls
root@820f7b697e1e:/home#
```

when I try ssh in that field:
![Zrzut ekranu_20220211172228](https://user-images.githubusercontent.com/25219988/153629023-70fe0cd2-4781-4149-b54e-1302ff79a325.png)
I have that error in another browser window:
```
disconnected
failed connecting to node 192.168.0.19. Teleport proxy failed to connect to "node" agent "192.168.0.19:3022" over reverse tunnel:

  no tunnel connection found: no node reverse tunnel for  found

This usually means that the agent is offline or has disconnected. Check the
agent logs and, if the issue persists, try restarting it or re-registering it
with the cluster.
```
Can You explain me easy way what happening here?
Regards.

**Reply by scottpgallagher** [reply]
Does the dietpi user exist on the system? if you want to create the user upon login you would need to look into our PAM module: https://goteleport.com/docs/server-access/guides/ssh-pam/#create-local-unix-users-on-login

**Reply by PrzemekSkw** [reply]
Hello, I find Teleport too complicated for me. I remove my docker installation. I'm looking for something easy like Guacamole, just to connect to my servers with ssh from web browser.
That discussion can be closed 
Regards.

**Comment by scottpgallagher** [comment]
@PrzemekSkw sorry to hear your having issues. I wouldn't recommend starting out with Docker/Docker Compose to get a cluster up and running to test. The easiest method would be to use Terraform to do so (https://github.com/gravitational/teleport/tree/master/examples/aws/terraform) or download the installer (https://goteleport.com/teleport/download/) and follow the docs to get started with (https://goteleport.com/docs/getting-started/linux-server/)

**Reply by PrzemekSkw** [reply]
@scottpgallagher thanks for still trying to help me but Teleport isn't project for me. I'm amateur who want to connect to my devices with ssh and sftp. Installation process isn't problem. I don't understand Teleport dashboard. There's no easy way to connect. Like type IP, port and password to connect  like in other ssh clients.
Regards.
